demos/PersonalizerBusinessDemo/PersonalizerBusinessDemo/Views/Home/Index.cshtml

```diff
@@ -327,6 +327,14 @@ In this demo, reward is computed by seeing how far down the user scrolls. This r
     </div>
 </footer>
 
+<div class="row my-2">
+    <div class="mx-auto">
+        <div class="row">
+            <span class="mx-auto" style="font-size:14px;"><i class="far fa-copyright"></i> 2019 - Demo of Personalizer, an Azure Cognitive Service <a href="https://privacy.microsoft.com/en-us/PrivacyStatement" target="_blank">Privacy</a></span>

```

Comment:
Have you tested that fixed font size in all resolutions?

Reply:
Yep